### ISSUE TITLE
argon2-browser not resolved

### ISSUE THREAD

**Issue by aryzing** [import-js/eslint-plugin-import#3207]
When importing like so:

```ts
import * as argon2 from 'argon2-browser';
```

a linting error is thrown:

```
No exported names found in module 'argon2-browser' (import/namespace)
```

yet it seems there are several exported names. VS Code successfully finds them with go to definition.

Is the top level empty `export { }` in the [type definitions](https://www.npmjs.com/package/@types/argon2-browser) throwing off this plugin?

**Comment by ljharb** [comment]
https://app.unpkg.com/argon2-browser@1.18.0/files/lib/argon2.js is a UMD file, so it doesn't actually have any named exports. In other words, you should be doing `import argon2 from 'argon2-browser'`.

(make sure, while using TS, you're using esModuleInterop and synthetic imports, or else TS's module system is broken)
